lib/components/Button/Button.treat.ts

```diff
@@ -108,12 +108,14 @@ export const variant = {
 export const defaultStates = {
 	primary: style(theme => ({
 		':hover': {
+			color: theme.colours.gamut.white,
```

Comment:
This should also be applied to all other button variants also?

Reply:
Done